packages/plotly/src/scatterplot.ts

```diff
@@ -6,65 +6,46 @@ import {
 import { IOnboardingScatterplotSpec } from '@visahoi/core/src/scatterplot'
 
 function extractOnboardingSpec (chart: any, coords): IOnboardingScatterplotSpec {
-  const traceNodes = chart.querySelectorAll('g.points')
-  const areaNodes = traceNodes[0].querySelectorAll('path.point')
-  const areaNodesData = Array.from(areaNodes).map(
-    (point: any) => point.__data__
-  )
-  const t = areaNodesData[0].trace
-
-  const grid = document
-    .getElementsByClassName('nsewdrag drag')[0]
-    .getBoundingClientRect()
-
-  const points = (Array.from(areaNodes) as any[]).filter(
-    (point) =>
-      point.getBoundingClientRect().x &&
-      point.getBoundingClientRect().x <= grid.width + grid.x &&
-      point.getBoundingClientRect().y &&
-      point.getBoundingClientRect().y <= grid.height + grid.y
-  )
-
-  const xVals = points.map((point) => point.getBoundingClientRect().x)
-  const yVals = points.map((point) => point.getBoundingClientRect().y)
-
-  const maxX = Math.max(...xVals)
-  const maxXIndex = xVals.indexOf(maxX)
-  const maxY = yVals[maxXIndex]
```

Comment:
why did you remove the min and max values? Those would be nice to annotate, at least for the basic messages.